### PR TITLE
Expose names for bundled container images

### PR DESCRIPTION
**Pull request by kasperl**
It is a little bit inconvenient that the names of dynamically installed container images, do not show up. I'll think about a good way of getting them included, but it is hard to modify the system image assets and there are downsides (mostly size) to encoding the names of the images as assets associated with the individual images.

Jaguar already maintains names for the dynamically installed ones. Not sure if we can use that map somehow or if we need to find a way to provide a name when container images are installed.

### CODE REVIEW

**Comment by kasperl** [comment]
Looks like this needs a bit more work (failing test).

File: lib/system/api/containers.toit [hunk 1]
```diff
@@ -47,9 +47,9 @@ class ContainerServiceClient extends ServiceClient implements ContainerService:
 
   list_images -> List:
     array := invoke_ ContainerService.LIST_IMAGES_INDEX null
-    return List array.size / 3:
-      cursor := it * 3
-      ContainerImage (uuid.Uuid array[cursor]) array[cursor + 1] array[cursor + 2]
+    return List array.size / 4:
+      cursor := it * 4
+      ContainerImage (uuid.Uuid array[cursor]) array[cursor + 1] array[cursor + 2] array[cursor + 3]
```
Comment (floitsch) [inline lib/system/api/containers.toit:52]: Would be nice if `ContainerImage` used named parameters.
That would immediately tell us what all these values are.

**Review by floitsch — APPROVED** [review]
LGTM.

File: lib/system/api/containers.toit (hunk 1 above)
Comment (kasperl) [reply lib/system/api/containers.toit]: Done.

**Pull request merged**